hack/run-image-scan.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+success_images=()
+failed_images=()
+
+if [ "$#" -lt 1 ] || [ "${1}" == "--help" ]; then
+  cat << EOF
+Usage:
+  $(basename $0) -i image1,image2
+  $(basename $0) -f fs1,fs2
+Example:
+  $(basename $0) -i quay.io/kubermatic/kubermatic:v2.21.1,quay.io/kubermatic/kubermatic:v2.21.0
+  $(basename $0) -f https://github.com/kubermatic/kubeone
```

Comment:
I think that `-f` is supposed to take a path on local file system (if we're talking about file system scanning).

Reply:
Yes, you are right. Will update that